mmvb_backend/benchmarking_sessions/tasks.py

```diff
@@ -148,8 +148,20 @@ def run_benchmark(self, benchmarking_session_id):
 
                 ai_response = response.json()
 
+                if "error" in ai_response:
+                    reporter.error(
+                        ai_implementation.id,
+                        BenchmarkingStepError.SERVER_ERROR,
+                    )
+                    continue
+
                 # todo: implement proper validation of response
-                if ai_response.get("triage", "") not in ["SC", "PC", "EC"]:
+                if ai_response.get("triage", "") not in [
```

Comment:
I believe there is a definition for the triage options now:
`from common.definitions import TRIAGE_OPTIONS`
then
`if ai_response.get("triage", "") not in TRIAGE_OPTIONS:`